### DISCUSSION TITLE
How to remove title space

### DISCUSSION THREAD

**Discussion by Kamo1966** [nhn/tui.chart#717] [Q&A] [answered]
Hallo, is there a way to remove space used for title at the top of graph?
I'm using this options in a responsive setting to remove extra space but the space used by title in the top of graph remains:
        options: {
          chart: { title: '' },
          legend: {
            visible: false
          },
          exportMenu: {
            visible: false
          }
        }

Thanks for your great job

**Comment by jwlee1108** [comment, marked as answer]
I'm sorry I missed this question. I didn't mean to.

To remove the top space as much as possible, delete the `title` option and set the `exporMenu`s `visible` option to `false`.
You mentioned `responsive`, so I left the relevant example option below, please refer to it.
```js
options: {
    chart: {
      width: 'auto',
      height: 'auto',
    },
    exportMenu: { visible: false },
    responsive: {
      animation: { duration: 300 },
    },
}
```

**Reply by jwlee1108** [reply]
Instead of `{title: ''}`, you need to erase it.

```js
options: {
    chart: {}, // remove `title: ''`
    exportMenu: {
        visible: false
    }
}
```

**Reply by Kamo1966** [reply]
Ok, fixed
you need to change your example code in responsive feature page (https://github.com/nhn/tui.chart/blob/main/docs/en/common-responsive-options.md): 
please change in this piece of code the title settings :
..
        condition: ({ width: w, height: h }) => {
          return w <= 500 && h <= 400;
        },
        options: {
          chart: { title: '' },
          legend: {
..
to remove extra space you have to change this:
chart: { title: '' } -> chart: { title: null }
thank you for your replay

**Reply by jwlee1108** [reply]
Thank you for your suggestion.

The meaning of `title: ''` in the example does not mean to get rid of a title or area. 
But I think other people can be confused like you, so I'll change the example by adding some string.
